server/services/AnomalyDetectorService.js

```diff
@@ -42,6 +42,42 @@ export default class DestinationsService {
     }
   };
 
+  validateDetector = async (req, h) => {
+    const params = { body: JSON.stringify(req.payload.configs) };
+    const { callWithRequest } = this.esDriver.getCluster(CLUSTER.AD_ALERTING);
+    try {
+      const resp = await callWithRequest(req, 'alertingAD.validateDetector', params);
+      return {
+        ok: true,
+        response: resp,
+      };
+    } catch (err) {
+      console.error('Alerting - AnomalyDetectorService - validateDetector:', err);
+      return { ok: false, resp: err.message };
+    }
+  };
+
+  createDetector = async (req, h) => {
+    const { callWithRequest } = this.esDriver.getCluster(CLUSTER.AD_ALERTING);
+    const requestBody = { body: JSON.stringify(req.payload.configs) };
+    let params = {
+      detectorId: undefined,
+      ifSeqNo: undefined,
+      ifPrimaryTerm: undefined,
```

Comment:
These should be determined by the backend, shouldn't need to be declared as undefined here